oci/spec_freebsd.go

```diff
@@ -0,0 +1,40 @@
+package oci
+
+import (
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func defaultMounts() []specs.Mount {
+	return []specs.Mount{
+                {
+                        Destination: "/proc",
+                        Type:        "procfs",
+                        Source:      "proc",
+                        Options:     []string{"nosuid", "noexec"},
+                },
+                {
+                        Destination: "/dev",
+                        Type:        "devfs",
+                        Source:      "devfs",
+                        Options:     []string{},
```

Comment:
Similarly  here do we not need "strictatime" in defaults?

Reply:
strictatime also isn't a supported mount option on FreeBSD